lib/ansible/plugins/lookup/password.py

```diff
@@ -311,7 +295,10 @@ def run(self, terms, variables, **kwargs):
             # make sure only one process finishes all the job first
             first_process, lockfile = _get_lock(b_path)
 
-            content = _read_password_file(b_path)
+            content = None
+            if os.path.exists(b_path):
+                b_content, show_data = self._loader._get_file_contents(b_path)
```

Comment:
Could you please add a change log entry [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments) and test for this change? Thanks.

Reply:
Added the change log entry, but I have a hard time getting the test to work:
- I'm new to python
- There are no existing tests for the `file` lookup